CMakeLists.txt

```diff
@@ -35,9 +35,10 @@ option(REFDM_UI_CLI "Enable text UI CLI for refdm" OFF)
 option(BUILD_TESTING "Enable test fixtures and libraries" OFF)
 option(TEST_MEMCHECK "Enable test runtime memory checking" OFF)
 option(TEST_COVERAGE "Enable test runtime coverage logging" OFF)
+option(IS_SUBDIRECTORY "Is a subidrectory of another project. Do not build tests." OFF)
```

Comment:
Another PR #159 adds separate option `BUILD_UNITTEST`. Would that be better suited for this purpose?